pkg/plugins/policies/meshtrafficpermission/api/v1alpha1/validator.go

```diff
@@ -39,6 +39,7 @@ func validateFrom(from []From) validators.ValidationError {
 				common_api.MeshSubset,
 				common_api.MeshService,
 				common_api.MeshServiceSubset,
+				common_api.MeshExternalService,
```

Comment:
Why do we allow setting `MeshExternalService` in from section for MTP?

Reply:
@lukidzi help me out here but I think otherwise egress won't work

Reply:
Yes, it's for an egress.